update the doc for the new log file locations (#96)
learningequality/kolibri-docs: docs/manage/_troubleshooting.rst

Locate Kolibri log files
------------------------

When you report a problem with Kolibri, we may ask you to send us Kolibri **log** files to help us find out why is it not working or crashing.

> Open the ``.kolibri`` folder inside the :ref:`Home <home>` folder of the device where Kolibri is running and locate the ``logs`` folder. You will be able to find these two files:

Reviewer: ```suggestion
Open the ``.kolibri/`` folder inside the :ref:`Home <home>` folder of the Kolibri server and locate the ``logs/`` folder. You will be able to find these two files:
```

Author: wow this is cool github feature! thank you for the suggestions!

Reviewer: Yeah, it's nice -- I try to only use it for documentation and the likes, since it's a bit more tricky for code and syntax stuff :)

* ``kolibri.txt``
* ``debug.txt``

> If the problem happened at a certain date, you can open the ``archive`` folder under the ``logs`` folder to find the specific log file:

Reviewer: ```suggestion
If the problem happened earlier than the dates in the above log, you can open the ``archive/`` folder inside ``logs/`` to find older log files:
```
